src/views/profile/bookmarks-view/components/empty-state/empty-state.module.css

```diff
@@ -0,0 +1,22 @@
+.wrapper {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	border: 1px dashed var(--token-color-neutral-border-strong);
```

Comment:
I chose to not implement a custom 'dashed' border to fully match the spec for skateboard mode.